examples/crawler.go

```diff
@@ -1,29 +1,24 @@
 package main
 
-// NOTE: While the task is good for driving API design, this
-// implementation is - by design - sequential, and can't make
-// use of the concurrency management offered by scheduler. Instead,
-// to better demonstrate the use case, we should have a second job
-// which gets the name of the first commenter for every post.
-
 import (
+  "io"
+  "os"
   "log"
-  "sync"
+  "fmt"
   "errors"
   "net/http"
   "github.com/PuerkitoBio/goquery"
- // "github.com/hidden-alphabet/scheduler"
+
   ".."
```

Comment:
Optionally, this should be replaced with " "github.com/hidden-alphabet/scheduler" to be more 'real-world'.
```suggestion
   "github.com/hidden-alphabet/scheduler"
```